app/bundles/LeadBundle/Translations/en_US/messages.ini

```diff
@@ -261,6 +261,8 @@ mautic.lead.lead.events.campaigns.expression="Expression"
 mautic.lead.lead.events.campaigns.date="Date"
 mautic.lead.lead.events.owner="Contact owner"
 mautic.lead.lead.events.owner_descr="Condition based on a contact owner."
+mautic.lead.lead.events.condition_donotcontact="Has flag DNC"
```

Comment:
Just thinking about better naming.
Has Do Not Contact flag
Maybe @RCheesley as native speaker could take look on it.

Reply:
I would suggest 'Marked as DNC' if we are trying to show why a contact is not being contacted, their DNC status or to confirm the successful application of the DNC status.

Reply:
that sounds perfect 👍